### DISCUSSION TITLE
模板编辑器中部分模板变为“undefined”

### DISCUSSION THREAD

**Discussion by Cateyes123** [windingwind/zotero-better-notes#846] [Q&A] [answered]
之前安装了一次Zotero7，后面又重装回Zotero6了，重装后出现部分模板Undefined，如图所示。请问我应该如何删除这些模板，双击右键无反应，删除键无法点击
![截图](https://github.com/windingwind/zotero-better-notes/assets/142150863/9f53e73e-9b23-4f4b-8776-8bd5e6d1fed9)
电脑版本：Win10 22H2
Zotero：6.0.30
插件：1.0.4

**Comment by windingwind** [comment, marked as answer]
https://github.com/windingwind/zotero-better-notes/discussions/802

善用搜索

**Reply by Cateyes123** [reply]
好的，非常感谢！
